test/integration/lib/tx-list-items.js

```diff
@@ -1,67 +0,0 @@
-const reactTriggerChange = require('../../lib/react-trigger-change')
```

Comment:
This test was deleted because it didn't test anything not already covered by e2e tests